geoextent/lib/handleCSV.py

```diff
@@ -1,12 +1,15 @@
 import csv
-import os
 import logging
 from . import helpfunctions as hf
 
 fileType = "text/csv"
 
 logger = logging.getLogger("geoextent")
 
+search = { "longitude" : ["(.)*longitude","(.)*long(.)*", "^lon","lon$","(.)*lng(.)*", "^x","x$"],
```

Comment:
Please add documentation for the supported options in the user docs!

Reply:
Ok ! I'm also including documentation regarding datetime format.